src/cascadia/WindowsTerminal/resource.h

```diff
@@ -3,20 +3,31 @@
 // Used by WindowsTerminal.rc
 //
 #define IDI_APPICON 101
-#define IDS_ERROR_DIALOG_TITLE 105
-#define IDS_HELP_DIALOG_TITLE 106
-#define IDS_ERROR_ARCHITECTURE_FORMAT 110
-#define IDS_X86_ARCHITECTURE 111
-#define IDS_AMD64_ARCHITECTURE 112
-#define IDS_ARM64_ARCHITECTURE 113
-#define IDS_ARM_ARCHITECTURE 114
-#define IDS_UNKNOWN_ARCHITECTURE 115
+#define IDI_APPICON_HC_B 102
+#define IDI_APPICON_HC_W 103
+
+#define IDI_APPICON_DEV 104
+#define IDI_APPICON_DEV_HC_B 105

```

Comment:
btw, maybe also update .\Generate-TerminalAssets.ps1 to automate the creation of the win32 assets (terminal-Pre_contrast-black.ico et al).